parser/model/model.go

```diff
@@ -1836,17 +1836,52 @@ type ResourceGroupRefInfo struct {
 // ResourceGroupSettings is the settings of the resource group
 type ResourceGroupSettings struct {
 	RURate           uint64 `json:"ru_per_sec"`
+	Priority         uint64 `json:"priority"`
 	CPULimiter       string `json:"cpu_limit"`
 	IOReadBandwidth  string `json:"io_read_bandwidth"`
 	IOWriteBandwidth string `json:"io_write_bandwidth"`
 	BurstLimit       int64  `json:"burst_limit"`
 }
 
+// NewResourceGroupSettings creates a new ResourceGroupSettings.
+func NewResourceGroupSettings() *ResourceGroupSettings {
+	return &ResourceGroupSettings{
+		RURate:           0,
+		Priority:         MediumPriorityValue,
+		CPULimiter:       "",
+		IOReadBandwidth:  "",
+		IOWriteBandwidth: "",
+		BurstLimit:       0,
+	}
+}
+
+// PriorityValueToName converts the priority value to corresponding name
+func PriorityValueToName(value uint64) string {
+	switch value {
+	case LowPriorityValue:
+		return "LOW"
+	case MediumPriorityValue:
+		return "MEDIUM"
+	case HighPriorityValue:
+		return "HIGH"
+	default:
+		return "MEDIUM"
```

Comment:
Does it mean, expect the 1,16, others will be handled as `MEDIUM`.

Reply:
yes